packages/react/src/themes/teams/components/Tooltip/tooltipContentStyles.ts

```diff
@@ -0,0 +1,37 @@
+import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
+import { TooltipContentProps } from '../../../../components/Tooltip/TooltipContent'
+import { TooltipContentVariables } from './tooltipContentVariables'
+import getPointerStyles from '../../getPointerStyles'
+
+const tooltipContentStyles: ComponentSlotStylesInput<
+  TooltipContentProps,
+  TooltipContentVariables
+> = {
+  root: ({ props: p, variables: v, rtl }): ICSSInJSStyle => ({
+    borderRadius: v.borderRadius,
+    display: 'block',
+    maxWidth: v.maxWidth,
+
+    ...(p.pointing && getPointerStyles(v.pointerOffset, v.pointerMargin, rtl, p.placement).root),
```

Comment:
May be two separate functions will be better, for `root` and `pointer`?

Reply:
This is how getBorderFocusStyles is implemented, so I wouldn't change this one. Btw this is just extracted as a separate function, so wouldn't like to change it as part of this PR.